Set up base data provider (#11)
businext/businext.ai: README.md

# businext.ai

**Bus**iness **in**sight **ext**ractor **a**nd **i**nterpreter?

# Configurations

> These configurations are to set the api keys for the various of services that are used to fetch images and data from. These are to be set in a `.env` file in the root directory of the project.

Reviewer: ```suggestion
These configurations are to set the api keys for the various services that are used to fetch images and data from. These are to be set in a `.env` file in the root directory of the project.
```

| Environment Variable Name | Value Format |                                 Example                                  | Explanation                                                                                       |
| ------------------------- | :----------: | :----------------------------------------------------------------------: | ------------------------------------------------------------------------------------------------- |

> | `BUSINESS_API_CONFIG`     |     JSON     | Click [here](README.md#business_api_config-example) to go to the example | Stringified Configuration object which contains your confidential information to access the APIs. |

Reviewer 2: ```suggestion
| `BUSINESS_API_CONFIG`     |     JSON     | Click [here](README.md#business_api_config-example) to go to the example | Serialized Configuration object containing your confidential information to access the APIs. |
```

> | `BUSINESS_API_CONFIG`     |     JSON     | Click [here](README.md#business_api_config-example) to go to the example | Stringified IConfiguration object which contains your confidential information to access the APIs. |

Reviewer 2: ```suggestion
| `BUSINESS_API_CONFIG`     |     JSON     | Click [here](README.md#business_api_config-example) to go to the example | Stringified Configuration object which contains your confidential information to access the APIs. |
```

Reviewer 2: will also need to rerun Prettier

Author: `Configuration` will be changed to a separate name.

> # `BUSINESS_API_CONFIG` Example

Reviewer 3: ```suggestion
## `BUSINESS_API_CONFIG` Example
```
